inference/core/managers/metrics.py

```diff
@@ -99,3 +102,36 @@ def get_inference_results_for_model(
         inference_results.append({"request_time": score, "inference": result})
 
     return inference_results
+
+
+def get_container_stats(socket_name: str = "/var/run/docker.sock"):
```

Comment:
I'm not sure if we need docker socket, but general advise regarding docker.sock is that if container is compromised then with access to `docker.sock` attacker gains immediate access to host's root.